compiler/test/formatter_outputs/aliases.gr

```diff
@@ -23,15 +23,15 @@ type AReallyReallyReallyReallyReallyReallyReallyReallyReallyLongLineBreakingType
   b
 > = String
 
-type Foo<
+type Foo3<
   a, // Test comment
   b
 > = String
 
-type Foo<a, /* Test comment */ b> = String
+type Foo4<a /* Test comment */, b> = String
```

Comment:
this shouldn't have a trailing comma 🤔

Reply:
It doesn't, that would be 
`type Foo4<a /* Test comment */, b, > = String`

Reply:
my eyes haha